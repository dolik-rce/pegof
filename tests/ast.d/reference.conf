input ast.d/reference.peg
ast
