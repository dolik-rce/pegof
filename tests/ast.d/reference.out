GRAMMAR
  RULE X
    ALTERNATION
      SEQ
        TERM
          REF Y


  RULE Y
    ALTERNATION
      SEQ
        TERM
          STRING Y
